sdk/.prettierc.js

```diff
@@ -0,0 +1,16 @@
+const modifiedConfig = {
```

Comment:
I'm not sure but vscode may have problems reading `.prettierc.js` in case there is no on top level of project struct. IMO we can move it to `./` from `./sdk`